### ISSUE TITLE
Bwl Hc Nefarian Random AOE cast Shadowflame  and oneshots all when is cast with Repelling Wave

### ISSUE THREAD

**Issue by dieguitoxu** [Duskhaven/Bug-Reports#1416]
We fight agains Nef 2 consecurive times, and got kill by shadowflame when we are not in boss face.
![imagen](https://user-images.githubusercontent.com/132944604/236911599-8e3f64b4-0937-4702-96df-d12dff40cfcc.png)
![imagen](https://user-images.githubusercontent.com/132944604/236911909-3e91bdbc-a61a-4400-9daa-f589c8f8256d.png)
Random at 2 diferent boss time.
Also, some boses like Ebonroc undertuned, he doesnt cast shadow bolt volley anymore.

**Comment by Chromie-the-gnome** [comment]
Yeah nefarian overlapping abilities doesnt feel like a real fight. Its basically have rng by your side.
